prusti-viper/src/encoder/pure_function_encoder.rs

```diff
@@ -491,6 +491,7 @@ impl<'p, 'v: 'p, 'tcx: 'v> PureFunctionBackwardInterpreter<'p, 'v, 'tcx> {
                     place.local,
                     &place.projection[..],
                 ).with_span(span)?;
+                let encoded_place = encoded_place.try_into_expr().with_span(span)?;
```

Comment:
This is fine for now, but you'll need to implement for `PureFunctionEncoder` the analogous version of `ProcedureEncoder::postprocess_place_encoding` in order to encode array operations in pure functions and specifications.

Reply:
hm, i guess i should do at least the same wrapping with a local version of `encode_place` and introduce a `postprocess_place_encoding` function in `PureFunctionEncoder` as well; the current way with just `try_into_expr` sprinkled everywhere doesn't seem to be clean nor meant to stay fwiw.